Resources/doc/reference/form_types.rst

```diff
@@ -100,6 +100,13 @@ btn_add, btn_list, btn_delete and btn_catalogue:
 
     If you need to use a sortable ``sonata_type_model`` check the :doc:`../cookbook/recipe_sortable_sonata_type_model` page.
 
+.. note::
+
+    When using ``sonata_type_model`` with ``btn_add``, a jQuery event will be
+triggered when a child form is added to the DOM
+(``sonata-admin-setup-list-modal`` by default and
+``sonata-admin-append-form-element`` when using ``edit:inline``).
```

Comment:
when looking at the preview the lines are properly indented but there's a blank line between 105 and 106